mlsshared.sty

```diff
@@ -189,7 +189,7 @@
   breaklines=true,                 % automatic line breaking only at white-space
   keepspaces,                      % don't remove space such as those after closing parenthesis
   captionpos=b,                    % sets the caption-position to bottom
-  commentstyle=\color{commentcolor}\sffamily\small, % For the LaTeXML build this is smaller than for 'basicstyle', need due to sans serif font.
+  commentstyle=\color{commentcolor}\sffamily\smallifpdf, % For the LaTeXML build this is smaller than for 'basicstyle', need due to sans serif font.
```

Comment:
Isn't it better to simply remove the size command here and let the size be determined by the `basicstyle`?

Reply:
Yes.

Reply:
And that also means removing the comment.

Reply:
As far as I can tell it actually works, and I removed the comment as well.

Reply:
Looks good now.  Resolving.